Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -14,14 +14,18 @@ public class Game {
 	}
 
 	public func playGame() {
+		let maxNumberOfGuesses = guessManager.maxNumberOfGuesses
+		view.promptNumberOfGuesses()
+		let numberOfGuesses = view.receiveNumberOfGuesses(maxNumberOfGuesses)
```

Comment:
I beg to differ...any game setup should occur before you start playing the game.  Think of when you play an XBox game or something...you can choose number of players, and whatever other options...all before you hit start.  Not all user interaction means that the game is being played